.github/workflows/release-gem.yml

```diff
@@ -23,16 +23,12 @@ jobs:
 
       - uses: actions/checkout@v2
 
-      - uses: ruby/setup-ruby@v1
+      - uses: ruby/setup-ruby@v1.87.0
         with:
           ruby-version: 2.7
-          bundler-cache: true
-
-      - name: Build Gem
-        run: make gem
 
       - name: Release Gem
-        uses: cadwallion/publish-rubygems-action@master
+        uses: ciremoussadia/publish-rubygems-action@master
```

Comment:
Why do we need this one here while the original works perfectly fine on bmg, predicate, etc. ?

Reply:
@blambeau that one does not work when we have dependency on our make rules because of this https://github.com/cadwallion/publish-rubygems-action/blob/8f9e0538302643309e4e43bf48cd34173ca48cfc/entrypoint.sh#L21
I submitted a fix to the project but hasn't been merged yet. I asked on slack that we fork this in enspirit.

Reply:
Ok for making a fork on enspirit's github repo.